flake.nix

```diff
@@ -34,11 +34,24 @@
             enableDebug = true;
             runTests = true;
           });
-          crypto3-sanitize = (pkgs.callPackage ./crypto3.nix {
-            enableDebug = true;
+          crypto3-clang-sanitize = (pkgs.callPackage ./crypto3.nix {
+            stdenv = pkgs.llvmPackages_19.stdenv;
             runTests = true;
+            enableDebug = false;
             sanitize = true;
           });
+          crypto3-clang-sanitize-bench = (pkgs.callPackage ./crypto3.nix {
```

Comment:
why we need sanitize and bench at the same time?

Reply:
Removed